gravitee-apim-console-webui/src/management/configuration/notifications/notification-settings/notification-settings-list/environment-notification-settings-list.module.ts

```diff
@@ -0,0 +1,76 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import {
+  GioFormFocusInvalidModule,
+  GioFormHeadersModule,
+  GioIconsModule,
+  GioLoaderModule,
+  GioSaveBarModule,
+} from '@gravitee/ui-particles-angular';
+import { MatTableModule } from '@angular/material/table';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { UIRouterModule } from '@uirouter/angular';
+import { HttpClientModule } from '@angular/common/http';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { MatCardModule } from '@angular/material/card';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { ReactiveFormsModule } from '@angular/forms';
+
+import { EnvironmentNotificationSettingsListComponent } from './environment-notification-settings-list.component';
+
+import { EnvironmentNotificationsDetailsSettingsComponent } from '../notification-settings-details/environment-notifications-details-settings.component';
+import { GioGoBackButtonModule } from '../../../../../shared/components/gio-go-back-button/gio-go-back-button.module';
+import { GioPermissionModule } from '../../../../../shared/components/gio-permission/gio-permission.module';
+import { GioTableWrapperModule } from '../../../../../shared/components/gio-table-wrapper/gio-table-wrapper.module';
+import { NotificationSettingsListModule } from '../../../../../components/notifications/notifications-list/notification-settings-list.module';
+
+@NgModule({
+  declarations: [EnvironmentNotificationsDetailsSettingsComponent, EnvironmentNotificationSettingsListComponent],
+  exports: [EnvironmentNotificationsDetailsSettingsComponent, EnvironmentNotificationSettingsListComponent],
+  imports: [
+    CommonModule,
+    HttpClientModule,
+    UIRouterModule,
+    ReactiveFormsModule,
+    NotificationSettingsListModule,
+    GioIconsModule,
+    GioFormFocusInvalidModule,
+    GioFormHeadersModule,
+    GioGoBackButtonModule,
+    GioPermissionModule,
+    GioLoaderModule,
+    GioSaveBarModule,
+    GioTableWrapperModule,
+    MatButtonModule,
+    MatCardModule,
+    MatCheckboxModule,
+    MatDialogModule,
+    MatFormFieldModule,
+    MatIconModule,
+    MatInputModule,
+    MatSnackBarModule,
+    MatTableModule,
+    MatTooltipModule,
```

Comment:
Same I think we can clean unused imports here

Reply:
Done